### PR TITLE
Update feedback.png

### PR DESCRIPTION
**Pull request by msbemba**
Fixed the Screenshot for feedback

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23598

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender/feedback.md [hunk 1]
```diff
@@ -33,14 +33,10 @@ ms.date: 02/16/2021
...
-Your feedback helps us get better in protecting your environment from advanced attacks. Share your experience, impressions, and  requests by providing feedback.
+Your feedback helps us get better at protecting your environment from advanced attacks. Share your experience, impressions, and  requests by providing feedback.
 
-Check out this video to see how easy it is to provide feedback.
 
-> [!VIDEO https://www.microsoft.com/en-us/videoplayer/embed/RE4LWeP]
-
-
-1. From any part of the portal, select **Give feedback**. 
+1. From the home page of Microsoft Defender, select **Give feedback** and click on **Leave Feedback**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/feedback.md]:
```suggestion
1. From the Microsoft Defender home page, select **Give feedback** and then select **Leave Feedback**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder.

**Comment by Dansimp** [comment]
udpated in private PR. To be published soon.

**Pull request closed without merging**